### DISCUSSION TITLE
种子查询页面的“热门”标签是根据什么进行统计的？

### DISCUSSION THREAD

**Discussion by HaodongZou** [ronggang/PT-Plugin-Plus#1427] [Q&A] [answered]
<img width="562" alt="image" src="https://github.com/pt-plugins/PT-Plugin-Plus/assets/38043100/ccc193ed-7bd5-4bf2-b563-04b855d2f434">

这个“热门”标签下9个电影只有2个算得上热门，其他的要么是经典高分片，要么干脆是十分冷门的。比如这个“官人我要房”，各个站点都搜不到种子，且豆瓣评价人数都不足以展示评分，但是一直在“热门”下的第一位。

请问这个热门的数据是来源于哪里呢？感觉如果能优化一下，展示正常的数据就太好了。

**Comment by ted423** [comment, marked as answer]
https://github.com/pt-plugins/PT-Plugin-Plus/discussions/984

这个功能其实大部分时间都没法用
